test/GenericTest.java

```diff
@@ -0,0 +1,23 @@
+package test;
+
+import java.util.List;
+
+public class GenericTest {
```

Comment:
> Create class-container for any types;

There is nothing about animals. 
You have to define class-container (or class-wrapper) for any types. Like Cup from the presentation.